BOJ/01003-피보나치_함수/01003-피보나치_함수-yeonju.py

```diff
@@ -1 +1,12 @@
-# git commit -m "submit: BOJ 01003 피보나치 함수 (yeonju)"
\ No newline at end of file
+t = int(input())
+
+for tc in range(t):
+    n = int(input())
+    fibo = [0] * n
+    fibo = [(1, 0), (0, 1)] + [0] * (n - 1)
+
+    for i in range(2, n + 1):
+        fibo[i] = (fibo[i - 1][0] + fibo[i - 2][0], fibo[i - 1][1] + fibo[i - 2][1])
+
+    print(fibo[n][0], end=' ')
```

Comment:
```suggestion
    print(fibo[n][0], fino[n][1])
```
이거 띄어쓰기로만 구분할꺼면 end=" " 안하고 콤마로만 구분해도 됩니다!